client/mapreduce/src/main/java/org/apache/accumulo/core/client/mapred/AbstractInputFormat.java

```diff
@@ -337,7 +340,9 @@ protected static Authorizations getScanAuthorizations(JobConf job) {
    * @return The client configuration for the job
    * @since 1.7.0
```

Comment:
Could add deprecated since 2.0.0 to javadoc and point to replacement.

Reply:
Fixed